fedora-toolbox

```diff
@@ -56,7 +64,17 @@ create()
             fi
         fi
 
+        if $need_home_link ; then
+            if ! $prefix_sudo buildah run $working_container_name -- \
+                    sh -c 'rmdir /home && mkdir -m 0755 -p /var/home && ln -s var/home /home' 2>&42; then
+                $prefix_sudo buildah rmi $working_container_name >/dev/null 2>&42
```

Comment:
It should be `buildah rm`. It was a typo that was fixed in commit 62e20f9df3bea846. Sorry about that.